unitylibs/core/workflow/workflow-photoshop/workflow-photoshop.js

```diff
@@ -339,11 +339,13 @@ function createSlider(cfg, tray, propertyName, label, cssFilter, valObj) {
   const actionSliderCircle = createTag('a', { href: '#', class: `adjustment-circle ${propertyName}` }, actionAnalytics);
   actionSliderDiv.append(actionSliderInput, actionSliderCircle);
   actionDiv.append(actionLabel, actionSliderDiv);
+  const isRtl = document.documentElement.getAttribute('dir') === 'rtl';
```

Comment:
Nit: This can be simpler
```suggestion
  const isRtl = document.dir === 'rtl';
```